src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -125,22 +125,22 @@ export default function Swap({
   // Is fee greater than input?
   const { isFeeGreater, fee } = useIsFeeGreaterThanInput({ chainId, address: INPUT.currencyId, parsedAmount })
 
+  const toggledVersion = useToggledVersion()
```

Comment:
we should just remove this toggled version thing at some point but not relevant here

Reply:
Probably we should rethink this Mod and remove our custom code, and address these things